testnets/sgenetworktestnet/assetlist.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_id": "sge-network-3",
+  "assets": [
+    {
+      "description": "The SGE token is primarily a governance token for the SGE Network blockchain.",
+      "denom_units": [
+        {
+          "denom": "usge",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "usge",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "usge",
+      "name": "SGE Network",
+      "display": "sge",
+      "symbol": "SGE",
+      "logo_URIs": {
+        "png": "https://sgenetwork.io/images/sgenetwork-logo.png",
+        "svg": "https://sgenetwork.io/images/logo-SGE.svg"
+      }
+    },
+    {
+      "description": "The SGE token is primarily a governance token for the SGE Network blockchain.",
+      "denom_units": [
+        {
+          "denom":"ibc/CFDF212C4BE8349AEDF9DE5006F394717C7E9E814742E1D6DE95AE288F58054C",
+          "exponent": 0,
+          "aliases": [
+            "usge"
+          ]
+        },
+        {
+          "denom": "usge",
+          "exponent": 6
+        }
+      ],
+      "base":"ibc/CFDF212C4BE8349AEDF9DE5006F394717C7E9E814742E1D6DE95AE288F58054C",
+      "name": "SGE Network",
+      "display": "sge",
+      "symbol": "SGE",
+      "ibc": {
+        "source_channel": "channel-0",
+        "dst_channel": "channel-1015",
+        "source_denom": "usge"
+      },
+      "logo_URIs": {
+        "png": "https://sgenetwork.io/images/sgenetwork-logo.png",
```

Comment:
please point these to the local /testnets/sgenetworktestnet/images/ folder in this repo, and upload any necessary files.